source/GameData.cpp

```diff
@@ -136,6 +136,9 @@ future<void> GameData::BeginLoad(bool onlyLoadData, bool debugMode)
 		Music::Init(sources);
 	}
 
+	// Provide TestData with non-const access to the UniverseObjects so it can directly inject data.
+	TestData::ConfigureObjects(objects);
```

Comment:
The typical usage of GameData is to request the relevant dataset using a static function when it is needed. Not to intercept a pointer at load-time.
Can we also apply the typical usage in this PR? So that we keep a bit of uniformity in the way we access data?

Reply:
So, a public getter in GameData?